core/src/progress_map.rs

```diff
@@ -163,26 +163,22 @@ impl ValidatorStakeInfo {
 pub const RETRANSMIT_BASE_DELAY_MS: u64 = 5_000;
 pub const RETRANSMIT_BACKOFF_CAP: u32 = 6;
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub struct RetransmitInfo {
-    pub retry_time: Option<Instant>,
+    pub retry_time: Instant,
     pub retry_iteration: u32,
 }
 
 impl RetransmitInfo {
     pub fn reached_retransmit_threshold(&self) -> bool {
         let backoff = std::cmp::min(self.retry_iteration, RETRANSMIT_BACKOFF_CAP);
         let backoff_duration_ms = (1_u64 << backoff) * RETRANSMIT_BASE_DELAY_MS;
-        self.retry_time
-            .map(|time| time.elapsed().as_millis() > backoff_duration_ms.into())
-            .unwrap_or(true)
+        self.retry_time.elapsed().as_millis() > u128::from(backoff_duration_ms)
```

Comment:
this can just be...

```suggestion
        self.retry_time.elapsed() > backoff_duration_ms
```

if we switch to 
```rust
let backoff_duration_ms = Duration::from_millis((1_u64 << backoff) * RETRANSMIT_BASE_DELAY_MS);
```
above.  sry, github won't let me suggest the whole alternative :pensive: